platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py

```diff
@@ -993,7 +988,7 @@ def get_lpmode(self):
         """
         lpmode_state = False
         try:
-            if self.sfp_type.startswith('QSFP'):
+            if self.port_type == 'QSFP_DD':
```

Comment:
Make sure the power override bit is '0' in byte 93. Please see the truth table for Byte 93 in SFF 8636

Reply:
@prgeor I don't see power override byte is defined in CMIS spec. can you please point the truth table in CMIS spec if any?.

Reply:
Ok. I thought this is for QSFP. For QSFP-DD see the LowPwrS transition signal truth table Table 6-12 CMIS rev 5.0. Please note that the objective of get_lpmode() is not to read the HW pin corresponding to LPMode but to know if the module has entered Low power state. We can know module state by reading the Byte 3 Page 0h.